boards/nxp/frdm_mcxn236/doc/index.rst

```diff
@@ -68,6 +68,14 @@ The FRDM-MCXN236 board configuration supports the following hardware features:
 +-----------+------------+-------------------------------------+
 | WATCHDOG  | on-chip    | watchdog                            |
 +-----------+------------+-------------------------------------+
+| PWM       | on-chip    | pwm                                 |
++-----------+------------+-------------------------------------+
+| CTIMER    | on-chip    | counter                             |
```

Comment:
Put in a separate commit?

Reply:
Thanks for reminder, removed PWM and CTIMER.